src/Sarif.Converters/SemmleConverter.cs

```diff
@@ -114,6 +114,7 @@ private Result ParseResult()
             Region region = MakeRegion(fields);
             var result = new Result
             {
+                RuleId = fields[(int)FieldIndex.QueryName],
```

Comment:
Does the query name meet the requirements for a SARIF rule id? "... a stable, opaque identifier for the rule" Isn't it legal for the author of a Semmle query to rename it?